src/test/sql/sqlite_testrunner/sqlite_wrapper.cpp

```diff
@@ -155,7 +155,7 @@ void SQLiteWrapper::_create_columns(std::shared_ptr<Table> table, sqlite3_stmt*
         }
 
         case SQLITE_FLOAT: {
-          col_types[i] = "float";
+          col_types[i] = "double";
```

Comment:
?

Reply:
should have doc'ed that, sorry. SQLite's float columns are 64 bit, thus double.

Reply:
Maybe @lanice wants to shout in here, why he picked float, but I think it should be double